Zend/tests/bug26166.phpt

```diff
@@ -64,6 +64,6 @@ try {
 --EXPECT--
 Hello World!
 ===NONE===
-Method NoneTest::__toString() must return a string value
+Return value of NoneTest::__toString() must be of type string, none returned
```

Comment:
none vs nothing?

Reply:
unrelated to the PR

Reply:
Hmmm. @nikic what would be the impact of changing this globally?

Reply:
Lots of error messages changed in PHP 8, so feel free to tweak it.

Reply:
The impact could be breaking test suites for a subjective change IMHO...